lib/apiservers/engine/backends/container.go

```diff
@@ -58,6 +58,19 @@ import (
 
 const (
 	bridgeIfaceName = "bridge"
+
```

Comment:
I set the min to 1GiB for now.. although I think that still may be too little. @hickeng @caglar10ur what did you guys use for sensible minimums/defaults in Bonneville?